git-secrets

```diff
@@ -53,6 +53,18 @@ load_patterns() {
   done
 }
 
+# load patterns and combine them with |
```

Comment:
Unfortunately not, it did replace all whitespaces. This is problematic because a pattern could include a whitespace itself.